packages/element/src/create-interpolate-element.ts

```diff
@@ -26,25 +35,33 @@ let indoc, offset, output, stack;
  *
  * @type {RegExp}
  */
-const tokenizer = /<(\/)?(\w+)\s*(\/)?>/g;
+const tokenizer: RegExp = /<(\/)?(\w+)\s*(\/)?>/g;
```

Comment:
We can remove that `@type {RegExp}`. Also, you don't need to type annotate `tokenizer`, TS should be able to infer it. So, you may change to 

```ts
const tokenizer = /<(\/)?(\w+)\s*(\/)?>/g;
```